projects/1.16.1/assets/cazfps-the-dead-sea/cazfps_the_dead_sea/lang/zh_cn.json

```
@@ -0,0 +1,93 @@
+  "item.cazfps_the_dead_sea.deadclown_spawn_egg": "死亡小丑 刷怪蛋",
+  "block.cazfps_the_dead_sea.dtps": "死亡之树木台阶",
+  "block.cazfps_the_dead_sea.deadtreeslab": "死亡之树台阶",
+  "block.cazfps_the_dead_sea.handplant": "死之草",
+  "item.cazfps_the_dead_sea.oldman": "可怖老人 刷怪蛋",
+  "entity.cazfps_the_dead_sea.sirenhead": "塞壬之头",
+  "advancements.tff.title": "遗忘之森",
+  "block.cazfps_the_dead_sea.concretestairs": "混凝土楼梯",
+  "block.cazfps_the_dead_sea.steelstairs": "钢楼梯",
+  "block.cazfps_the_dead_sea.deadbush": "大型死之草丛",
+  "block.cazfps_the_dead_sea.deadstonebrick": "死之石砖",
+  "entity.cazfps_the_dead_sea.oldman": "可怖老人",
+  "item.cazfps_the_dead_sea.sirenhead_spawn_egg": "塞壬之头 刷怪蛋",
+  "block.cazfps_the_dead_sea.deadsandlayerbigg": "大层死之沙",
+  "item.cazfps_the_dead_sea.gunamo": "子弹",
+  "entity.cazfps_the_dead_sea.deadclown": "死亡小丑",
+  "advancements.deadclownw.descr": "gg!",
+  "block.cazfps_the_dead_sea.stair": "石路楼梯",
+  "block.cazfps_the_dead_sea.dsbslab": "死之石砖台阶",
+  "block.cazfps_the_dead_sea.concrete": "混凝土",
+  "block.cazfps_the_dead_sea.steelfence": "钢栅栏",
+  "block.cazfps_the_dead_sea.dtpw": "死之木板树墙",
+  "block.cazfps_the_dead_sea.deadstone": "死之石",
+  "block.cazfps_the_dead_sea.deadstonebrickwall": "死之石砖墙",
+  "block.cazfps_the_dead_sea.hangingbody": "吊尸",
+  "block.cazfps_the_dead_sea.concreteslab": "混凝土台阶",
+  "advancements.soa.title": "灰烬之街",
+  "block.cazfps_the_dead_sea.slabb": "石路台阶",
+  "block.cazfps_the_dead_sea.deadsandlayer": "死之沙层",
+  "block.cazfps_the_dead_sea.deadstonebrickstairs": "死之石砖楼梯",
+  "block.cazfps_the_dead_sea.back": "回到主世界",
+  "advancements.soa.descr": "欢迎！！",
+  "block.cazfps_the_dead_sea.deadstonewall": "死之石墙",
+  "block.cazfps_the_dead_sea.cross_2": "十字架之尸下部分",
+  "item.cazfps_the_dead_sea.deadwitch": "死亡女巫 刷怪蛋",
+  "item.cazfps_the_dead_sea.deadwitch_spawn_egg": "死亡女巫 刷怪蛋",
+  "advancements.tdg.title": "死亡花园",
+  "block.cazfps_the_dead_sea.deadstonefence": "死之石栅栏",
+  "entity.cazfps_the_dead_sea.forestghost": "死森幽魂",
+  "item.cazfps_the_dead_sea.gun": "枪",
+  "block.cazfps_the_dead_sea.steelblock": "钢块",
+  "item.cazfps_the_dead_sea.thedeadseadimension": "死亡之海维度传送门打火器",
+  "block.cazfps_the_dead_sea.cross_1": "十字架之尸上部分",
+  "item.cazfps_the_dead_sea.ghostwaipon": "cross_hearth",
+  "advancements.tdg.descr": "欢迎！！",
+  "advancements.tff.descr": "欢迎！！",
+  "block.cazfps_the_dead_sea.slabbbbb": "死之沙台阶",
+  "entity.cazfps_the_dead_sea.deadskeleton": "死亡骷髅",
+  "item.cazfps_the_dead_sea.deadskeleton_spawn_egg": "死亡骷髅 刷怪蛋",
+  "itemGroup.tabthedeadsea": "死亡之海",
+  "block.cazfps_the_dead_sea.deadstonebrickfence": "死之石砖栅栏",
+  "block.cazfps_the_dead_sea.roo": "金属屋顶",
+  "block.cazfps_the_dead_sea.deadtreestairs": "死亡之树楼梯",
+  "block.cazfps_the_dead_sea.deadtreewall": "死亡之树墙",
+  "item.cazfps_the_dead_sea.deadwalker_spawn_egg": "死亡游荡者 刷怪蛋",
+  "item.cazfps_the_dead_sea.sirenheadcall": "塞壬之首召唤器",
+  "block.cazfps_the_dead_sea.dtf": "死亡之树木板栅栏",
+  "item.cazfps_the_dead_sea.bigfoot": "大脚巨人 刷怪蛋",
```

Bigfoot是西方市井传说中的怪物，在中文语境下长期被译为大脚怪。建议从俗。
```suggestion
  "item.cazfps_the_dead_sea.bigfoot": "大脚怪 刷怪蛋",
```